crates/polars-time/src/group_by/dynamic.rs

```diff
@@ -659,6 +659,7 @@ fn update_subgroups_idx(
 #[cfg(test)]
 mod test {
     use chrono::prelude::*;
+    use polars_ops::prelude::abs;
```

Comment:
I'm not a huge fan of specifying precise dependencies through the prelude. Could you either import the proper path or import `*` from the prelude?